tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -1151,6 +1151,31 @@
         padding: 4px;
         margin-top: 4px;
       }
+      .pd-holder {
+        position: relative;
+        width: 300px;
+      }
+      .pd-y-label {
+        color: #5f6368;
+        font-size: 12px;
+        left: -20px;
+        padding: 0px;
+        position: absolute;
+        top: 80px;
+        transform: rotate(270deg);
+      }
+      .pd-x-label {
+        bottom: 0;
+        color: #5f6368;
+        font-size: 12px;
+        overflow: hidden;
+        padding: 0 0 0 40px;
+        position: absolute;
+        text-align: center;
+        text-overflow: ellipsis;
+        width: 100%;
+        white-space: nowrap;
```

Comment:
in the screenshot, I see wrapping of text in the x-axis. Is this referring to some other element?

Reply:
its for wrapping of the axes label (only showing on line charts). so in the screen shot it is "capital-loss". i checked with a very long string locally and showed correct ellipsis and non-wrapping behavior.

Reply:
Ah, got it. Thanks!